string skill_name
string launch_package
string launch_file
bool continuous
SkillNode[] node_list
---
int8 response
